wix-ecom-backend/wix-ecom-backend/guides/1-wix-ecom-backend-intro.md

```diff
@@ -3,27 +3,69 @@ title: Introduction
-| Inventory                                | [Wix Stores Backend](https://www.wix.com/velo/reference/wix-stores-backend)                                       | Wix Stores - no change
-| Collections                              | [Wix Stores Backend](https://www.wix.com/velo/reference/wix-stores-backend)                                       | Wix Stores - no change
+With the `wix-ecom-backend` Velo APIs, you can do the following and more:
 
-> **Note:** Further functionality will be coming soon.
++ Access, manage, and create a site visitor's [cart](https://www.wix.com/velo/reference/wix-ecom-backend/cart/introduction). The same functionality also applies, without the use of a cart ID, to a site visitor's [current cart](https://www.wix.com/velo/reference/wix-ecom-backend/currentcart/introduction).
++ Access, manage, and create a site visitor's [checkout](https://www.wix.com/velo/reference/wix-ecom-backend/checkout/introduction), and listen to an event when a checkout is [abandoned](https://www.wix.com/velo/reference/wix-ecom-backend/events/onabandonedcheckoutcreated) or [recovered](https://www.wix.com/velo/reference/wix-ecom-backend/events/onabandonedcheckoutrecovered).
++ Create and manage [discount rules](https://www.wix.com/velo/reference/wix-ecom-backend/discountrules/introduction).
++ [Promote and recommend](https://www.wix.com/velo/reference/wix-ecom-backend/recommendations/introduction) items to customers.
++ Create, access, and manage [orders](https://www.wix.com/velo/reference/wix-ecom-backend/orders), and retrieve information about order-related [transactions](https://www.wix.com/velo/reference/wix-ecom-backend/ordertransactions/introduction).
++ Manage [fulfillment](https://www.wix.com/velo/reference/wix-ecom-backend/orderfulfillments/introduction) (preparation and delivery) of orders.
+
+The Wix eCommerce platform also incorporates SPIs for greater customization. Use the SPIs to:
+
++ Incorporate custom [additional fees](https://www.wix.com/velo/reference/spis/wix-ecom/ecom-additional-fees/introduction).
++ Integrate [shipping rates](https://www.wix.com/velo/reference/spis/wix-ecom/ecom-shipping-rates/introduction).
++ [Validate](https://www.wix.com/velo/reference/spis/wix-ecom/ecom-validations/introduction) a site visitor's cart and checkout.
+
+The `wix-ecom-backend` Velo APIs are constantly being expanded with new capabilities. Check the API reference regularly to stay up to date with new functionality.
+
+## Before you begin
+
+The main difference between the Wix eCommerce and Wix Stores APIs is that while **Wix eCommerce manages the cart, checkout, and order** phases of the online store experience, **Wix Stores continues to manage product catalogs, inventories and collections**.
+
+We strongly recommend you migrate your code to the new `wix-ecom-backend` APIs and events, as the Wix Stores Cart and Orders APIs will be deprecated soon. The following tables outline the differences and required migrations between the two APIs:
```

Comment:
Consider being less apologetic. Most/many readers aren't familiar with Stores, so I think it makes sense to present the current division as "the way things are" and the migration information only for legacy code. Maybe something along these lines:

Wix eCommerce and Wix Stores handle different parts of the eCommerce flow for an online store:
+ **Wix eCommerce**: Carts, checkouts, and order.
+ **Wix Stores**: Product catalogs, inventories, and collections.

If your code still uses the legacy Wix Stores Cart and Orders APIs, we recommend you migrate them to the new Wix eCommerce APIs as soon as possible. The following tables outline the differences and required migrations: